requirements.txt

```diff
@@ -28,6 +28,7 @@ charset-normalizer>=3.1.0
 spacy>=3.0.0,<4.0.0
 en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.5.0/en_core_web_sm-3.5.0-py3-none-any.whl
 prompt_toolkit>=3.0.38
+langchain==0.0.205
```

Comment:
Do we want to pin LangChain to this specific version? They publish releases regularly, and profiting from those wouldn't be a bad thing imo.

Reply:
It's tricky because we also don't want a langchain update to unexpectedly break auto-gpt. I don't particularly care either way but this seems "safer"

Reply:
Agreed, and I can't find info on their versioning policy. This dependency is going to fall out of date *quickly* though, because as of now the latest release version of LangChain is v0.0.208: this PR is already behind by 3.